seleniumTest/stest_adminAccess.js

```diff
@@ -0,0 +1,23 @@
+var assert = require('assert');
+//var request = require('request');
```

Comment:
Did you get these tests to pass? 

These fail for me and this by design as you have protected the route. We shouldnt push broken test cases to master